Doom-Mummies/Welcome-to-Nashville
# Mj restaurants api

# Description

I got the restaurant search field to populate results when input is entered and search button is clicked. I also reformatted/reorganized html in searchForm.js for readability and spacing

Fixes # (issue)

## Type of change

Please delete options that are not relevant.



# Testing Instructions

1. ``git fetch --all``
2. ``git checkout mj-restaurantsApi``
3. please check that input into restaurant search field has a results output when search button is clicked.



# Checklist:
